### ISSUE TITLE
Config option for git --no-verify

### ISSUE THREAD

**Issue by patrik-piskay** [webpro/release-it#245]
Hi, we are using `release-it` for an automated github releases and we'd appreciate a config option to run `git commit` and `git push` cmds with the `--no-verify` option to skip the execution of git hooks (for code formatting/tests)

Would you be open to accepting a PR addressing this?

Thanks!

**Comment by webpro** [comment]
Hi @patrik-piskay. Good idea! And it probably makes sense to have something of a `src.commitExtraArgs` option or something, so you can put any Git args in there. Not sure whether that should be a string or an array.

An additional benefit is that we can also get rid of the global `force` option then.

**Comment by patrik-piskay** [comment]
Sounds good!

Would you prefer going for `src.commitExtraArgs` and `src.pushExtraArgs` (and for `dist` too probably?), or something like this?
```
src: {
  gitExtraArgs: {
    commit: [],
    push: []
  }
}
```

**Comment by webpro** [comment]
There's already `src.git.commit` and `push`, so I think `src.git.commitArgs: []` etc. (and also for `dist`) would be clear and concise. What do you think?

**Comment by webpro** [comment]
This can be closed as #247 was merged.
